Monika-After-Story/MonikaModDev
# Fixed retro-style ribbons having wrong acs type

fixes #7023 

# Key Changes
* changed the `acs_type` and names of the retro bows to retro ribbons - these are different colored ribbons, not bows.
     * NOTE: I did not change the `name` for these ACS so that they don't have to be re-gifted. 

# Testing
* Verify that the "bows" are now referred to as ribbons in-game
* Verify that you do not have re-gift these ribbons after applying the spritepack update after you have already gifted them when they were called "bows".
